server/src/main/java/io/druid/query/lookup/LookupModule.java

```diff
@@ -185,34 +194,44 @@ public LookupResourceListenerAnnouncer(
 class LookupListeningAnnouncerConfig extends ListeningAnnouncerConfig
 {
   public static final String DEFAULT_TIER = "__default";
+  private final DataSourceNameHolder dataSourceNameHolder;
   @JsonProperty("lookupTier")
   private String lookupTier = null;
```

Comment:
i am not sure, does this get its value from jackson even if it is final and had null on it ?